pkg/plugin/manifest/serializer.go

```diff
@@ -27,6 +27,12 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/serializer/json"
 )
 
+const (
+	container string = "container"
+	volume    string = "volume"
+	manifest  string = "manifest"
```

Comment:
Since these are actually being used as the `kind` can we rename them to be `kindManifest`, `kindVolume`, etc. So it is more clear in which contexts those constants are usable (as opposed to just making the english word a constant to avoid spelling mistakes or whatnot)